docs/how-to-guides/work-with-triggers.md

```diff
@@ -0,0 +1,84 @@
+---
+id: work-with-triggers
+title: How to work with triggers
+sidebar_label: Work with triggers
+---
+
+Memgraph supports **database triggers** that can be executed if a particular type of event occurs. Events are related to changes in data, such as created, updated, and deleted nodes or relationships. 
+
+[![Related - Reference Guide](https://img.shields.io/static/v1?label=Related&message=Reference%20Guide&color=yellow&style=for-the-badge)](/reference-guide/triggers.md) 
+
+
+## How to create a trigger?
```

Comment:
I think I would add subchapters here for three different types of triggers and treat each subchapter as an individual entity... so each has it's own intro what we are trying to do, example, and explanation:

H2 How to create a trigger?

Smal intro 2-3 sentences

H3 Trigger execution upon node creation

Why do I want it?
Example
Explanation

H3 Trigger execution upon node or relationship creation

Why do I want it? How is this different from the node creation
Example
Explanation

H3 Trigger execution upon node update

Why do I want it?
Example
Explanation

Reply:
It was my inital tought, how does it look now? @vpavicic